Watch Extension/NewsInterfaceController.swift

```diff
@@ -6,29 +6,29 @@
 //  Copyright © 2016 Ezequiel Becerra. All rights reserved.
 //
 
-import WatchKit
 import Foundation
+import WatchKit
 
 class NewsInterfaceController: WKInterfaceController {
-    
+
     var news: [News]?
-    
+
     @IBOutlet var newsTable: WKInterfaceTable!
-    
+
     override func awake(withContext context: Any?) {
         super.awake(withContext: context)
-        
+
         guard let dict = context as? [String: Any],
             let title = dict["title"] as? String,
             let news = dict["elements"] as? [News] else {
             return
         }
-        
+
         let sortedNews = news.sorted { $0.date.compare($1.date) == .orderedDescending }
 
         self.setTitle(title)
         self.newsTable.setNumberOfRows(sortedNews.count, withRowType: "NewsRow")
-        
+
         for (index, n) in sortedNews.enumerated() {
             let row = self.newsTable.rowController(at: index) as! NewsRowController
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Canillitapp/headlines-iOS/blob/f5a5ab936ee0d164c160af7374b637afef6eb821/Watch Extension/NewsInterfaceController.swift#L33"></span>Force casts should be avoided.<br />
<code>force_cast</code> <code>NewsInterfaceController.swift:33</code></td>
    </tr>
  </tbody>
</table>